tutoraspects/plugin.py

```diff
@@ -13,7 +13,6 @@
 from tutor import hooks
 
 from .__about__ import __version__
-from .commands_v0 import COMMANDS as TUTOR_V0_COMMANDS
```

Comment:
v0 commands were from earlier (pre-Palm?) versions or Tutor which we no longer support